data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2013,10 +2041,24 @@
             y=22-25
         [/filter]
 
-        [message]
-            speaker=unit
-            message= _ "Hey, an underground lake."
-        [/message]
+        [if]
+            [variable]
+                name=unit.id
+                equals="Krash"
+            [/variable]
+            [then]
+                [message]
+                    speaker="Krash"
+                    message= _ "Wet."
```

Comment:
Some of these could do with [notes to translators](https://wiki.wesnoth.org/GettextForWesnothDevelopers#Marking_up_strings_in_WML)

I suggest adding this line above the `message`:
```
# po: Krash has landed in one of the hexes of a small underground lake
```